features/matchers/have_broadcasted_matcher.feature

```diff
@@ -0,0 +1,151 @@
+@rails_post_6
+Feature: have_broadcasted matcher
+
+  The `have_broadcasted_to` (also aliased as `broadcast_to`) matcher is used to check if a message has been broadcasted to a given stream.
```

Comment:
This line should be broken up like the rest of our cukes.

Reply:
Done in https://github.com/rspec/rspec-rails/pull/2071/commits/86db9eb72fd42a24db3664cd5c4735b84dcfa59d